.github/actions/add-tag/index.js

```diff
@@ -0,0 +1,39 @@
+const github = require('@actions/github')
+const core = require('@actions/core')
+
+const {
+    GITHUB_TOKEN
+} = process.env
+
+const PR_LABEL = 'WEBONISE'
+const REPO_OWNER = 'segmentio'
+const REPO_NAME = 'analytics.js-integrations'
+
+const octokit = new github.GitHub(GITHUB_TOKEN)
+
+async function run () {
+  const { number, pull_request } = github.context.payload
+
+  // Check who submitted the PR
+  if (!pull_request.user.login.includes('vjnathe-webonise')) {
```

Comment:
Would prob be better to check for a branch or PR name pattern, such as `/SEG*`` rather than a person's name.

Reply:
Is it possible to check if the user is **not** a part of the 1-class-destinations Github team?

Reply:
Maybe this? https://developer.github.com/v3/teams/members/#get-team-membership